waitress/tests/test_task.py

```diff
@@ -202,6 +202,23 @@ def test_build_response_header_v11_200_no_content_length(self):
         self.assertEqual(inst.close_on_finish, True)
         self.assertTrue(('Connection', 'close') in inst.response_headers)
 
```

Comment:
@bertjwregeer I'm talking about branch coverage:  the PR does not add a new testcase for the `10x` status code, only for `204`.  It also doesn't update any existing testcases which might have triggered the same branch.